services/web/server/src/simcore_service_webserver/login/_2fa.py

```diff
@@ -100,9 +93,8 @@ async def send_sms_code(
     twilo_auth: TwilioSettings,
     twilio_messaging_sid: str,
     twilio_alpha_numeric_sender: str,
-    user_name: str = "user",
+    first_name: str,
```

Comment:
Should this maybe be a pydantic class to take care of the length, just like in the api-server. Same below

Reply:
yes, but one step at a time. I need to refactor users and groups plugin and logic before being able to do that.